policy.md

```diff
@@ -37,21 +37,22 @@ for Interoperabilty Solutions. This control helps Estonia protect
 trust in its (and other's) X-Road investments.
 
 
-**KEY RECOMMENDATION**: Secrecy doesn't equal security.  Open source
-provides conditions for stronger security, along with opportunities
-for collaboration and cost sharing to improve security. 
+**KEY RECOMMENDATION**: Do not rely on security through obscurity. Secrecy doesn't 
+equal security, while open source provides the conditions for stronger security. 
```

Comment:
Hmm, we shouldn't use a comparative without saying than what -- that is, "stronger security than X".  What is it stronger than?

Reply:
will revise to be more explicit that the comparison is against proprietary